app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,16 +1,34 @@
 // Import Google fonts
-@import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700|Geologica:700,800|Khula:400&display=swap'); // Ensure Khula Regular is included
 
 // Define fonts for body and headers
 $body-font: "Work Sans", "Helvetica", "sans-serif";
```

Comment:
Let's just change the body font to Khula, then we don't need the paragraph variable.

Reply:
Yes that makes sense!